odata-protocol/11.4 Data Modification.md

```diff
@@ -577,9 +577,9 @@ supporting this SHOULD advertise it by annotating the singleton with the
 term `Capabilities.UpdateRestrictions` (nested property `Upsertable`
 with value `true`) defined in [OData-VocCap](#ODataVocCap).
 
-Key and other non-updatable properties, as well as dependent properties
-that are not tied to key properties of the principal entity, MUST be
-ignored by the service in processing the Upsert request.
```

Comment:
The non-updatable etc. part is already covered in the sections on insert and update, and improvements to those sections are proposed in #1968